testnet-2/02a285496a665f04d8f7e40958c6dc70275131c931f1f50438592e70d9b272340a.json

```diff
@@ -1,10 +1,6 @@
 {
-  "id": 18,
-  "name": "",
-  "secp": "02a285496a665f04d8f7e40958c6dc70275131c931f1f50438592e70d9b272340a",
-  "bls": "afabeba3e0dc1eab2e0cf4b0f71c41fbed45f8a3601adc0d72e7921347216159a9a3354894b806637c2d9cd1209255ce",
```

Comment:
This fields `id`, `secp` and `bls` needs to be preserved. Can you please update your PR?